docs/source/tutorials/psij.ipynb

```diff
@@ -1,34 +1,23 @@
 {
  "cells": [
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Psij Getting Started Tutorial"
+    "### PSI/J-Python Getting Started Tutorial"
```

Comment:
and maybe keep this as a title - `# PSI/J-Python...`
(will be in sync with Matteo's PR)

Reply:
Not sure why the heading got changed there. Fixed. Thanks.